index.yaml

```diff
@@ -1773,38 +1773,6 @@ entries:
     urls:
     - assets/rancher-istio/rancher-istio-100.0.0.tgz
     version: 100.0.0
-  - annotations:
-      catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.32.100+up1.32.0
-      catalog.cattle.io/certified: rancher
-      catalog.cattle.io/display-name: Istio
-      catalog.cattle.io/namespace: istio-system
-      catalog.cattle.io/os: linux
-      catalog.cattle.io/provides-gvr: networking.istio.io.virtualservice/v1beta1
-      catalog.cattle.io/release-name: rancher-istio
-      catalog.cattle.io/requests-cpu: 710m
-      catalog.cattle.io/requests-memory: 2314Mi
-      catalog.cattle.io/ui-component: istio
-    apiVersion: v1
-    appVersion: 1.9.6
-    created: "2021-07-30T17:25:46.746643-07:00"
-    dependencies:
-    - condition: kiali.enabled
-      name: kiali
-      repository: file://./charts/kiali
-    - condition: tracing.enabled
-      name: tracing
-      repository: file://./charts/tracing
-    description: A basic Istio setup that installs with the istioctl. Refer to https://istio.io/latest/
-      for details.
-    digest: daab9ccb018dc9f0b9f13153307dc7d3c8dc961e44e8b85ca443fb17287a7343
-    icon: https://charts.rancher.io/assets/logos/istio.svg
-    keywords:
-    - networking
-    - infrastructure
-    name: rancher-istio
-    urls:
-    - assets/rancher-istio-1.9/rancher-istio-1.9.601.tgz
-    version: 1.9.601
```

Comment:
Uhh whats going on here, cc @brendarearden

Reply:
Not related to aks, splitting it into its own commit

Reply:
Not entirely sure what happened here. Chart version 1.9.601 is a chart in the 2.5 branch so it makes sense that this is being removed as part of this PR and I approve of this change becuase that asset does not exist in 2.6. 

I was working on changes to the 2.5 branch for a backport, so its possible that the chart asset was cached in one code editor, while I was working on the cluster-tool changes in the 2.6 branch in a different code editor which showed up in the generated index.yaml. I know we have had some other weirdness with the index.yaml so maybe that explains those situations as well..

Reply:
1.9.601 doesn't exist in any of the other branches (dev-v2.5, release-v2.5/6) so removing seems right.

Reply:
https://github.com/rancher/charts/pull/1381/files << it exists in my backport to 2.5 which is why I believe it was a caching issue. But I agree - the index.yaml for this aks operator bump looks accurate and good to merge!